spec/operations/workflows/unretire_subjects_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Workflows::UnretireSubjects do
+  let(:api_user) { ApiUser.new(build_stubbed(:user)) }
+  let(:workflow) { create(:workflow) }
+  let(:subject_set) { create(:subject_set, project: workflow.project, workflows: [workflow]) }
+  let(:subject_set_id) { subject_set.id }
+  let(:subject1) { create(:subject, subject_sets: [subject_set]) }
+  let(:subject2) { create(:subject, subject_sets: [subject_set]) }
```

Comment:
not blocking

same as above with the subject setup - you can setup multiple subjects with one call
```
let(:subject_set) do 
  create(:subject_set_with_subjects, num_subjects: 2, project: workflow.project, workflows: [workflow])
end
let(:subject_1) { subject_set.subjects.first }
let(:subject_2) { subject_set.subjects.last }
```